pylint/checkers/similar.py

```diff
@@ -451,7 +481,7 @@ def usage(status=0):
     print()
     print(
         "Usage: symilar [-d|--duplicates min_duplicated_lines] \
-[-i|--ignore-comments] [--ignore-docstrings] [--ignore-imports] file1..."
+[-i|--ignore-comments] [--ignore-docstrings] [--ignore-imports] [--ignore-signatures] file1..."
```

Comment:
Not related to this MR but I which this was done automatically by argparse or similar.

Reply:
Not clear

Reply:
Nothing to do on your part, juste talking to myself :)